packages/tocco-ui/src/EditableValue/typeEditors/TimeEdit.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import DurationEdit from './DurationEdit'
+import {calculateMilliseconds} from '../utils'
+
+const TimeEdit = props => {
```

Comment:
There is a bug in the component when i enter 18 into hour field and then 33 in minutes field the components calls onChange with value "{hoursOfDay: 6, minutesOfHour: 33}" instead of "{hoursOfDay: 18, minutesOfHour: 33}"